wikipedia-dark.user.css

```diff
@@ -122,7 +148,7 @@
   /*** Red links (non-existant pages) ***/
   div.vectorTabs li.new:not(.selected) a, .mw-plusminus-neg, a.new,
   #p-personal a.new {
-    color: #980000 !important;
+    color: #e33 !important;
```

Comment:
Isnt this a bit too bright?

Reply:
I was going for something that didn't just blend in with a dark theme, since redlinks are supposed to stand out a bit, and were harder to read with `#980000`. If there's another shade you'd like, I can swap it out.

Reply:
I dont mind really, just that e33 may burn some retinas...

Reply:
Yeah, we don't want it to be to obnoxious, how about a nice `#f11` then?

Reply:
Done. I think `#f11` is a good compromise here.